packages/fast-usdc/test/exos/advancer.test.ts

```diff
@@ -94,19 +99,30 @@ const createTestExtensions = (t, common: CommonSetup) => {
     },
   });
 
+  const mockBorrowerFacetCalls: unknown[][] = [];
+
   const mockBorrowerF = Far('LiquidityPool Borrow Facet', {
     borrow: (seat: ZCFSeat, amounts: { USDC: NatAmount }) => {
-      console.log('LP.borrow called with', amounts);
+      console.log('LpBorrowFacet.borrow called with', amounts);
+      mockBorrowerFacetCalls.push(['borrow', seat, amounts]);
```

Comment:
given that we're now actually consuming the args and checking them, is logging them still useful?

Reply:
Agree, removed